templates/components/form/item_virtualmachine.html.twig

```diff
@@ -0,0 +1,19 @@
+{% if has_host %}
+    <h3 class='text-start'>{{ __('List of hosts') }}</h3>
+{% else %}
+    <h3 class='text-start mt-5'>{{ __('List of virtualized environments') }}</h3>
+    {% if canedit %}
+        <div class='center firstbloc'>
+            <a class='btn btn-primary' href='{{ vmformurl }}'>
```

Comment:
Typically, I've been changing the list UI to show forms like this directly in the page with the list if they don't have a ton of fields.
If this remains as an anchor tag though, it needs `role="button"`.